lib/routes/account.js

```diff
@@ -1673,37 +1664,20 @@ module.exports = function (
       },
       handler: function (request, reply) {
         log.begin('Account.RecoveryEmailResend', request)
+        const email = request.payload.email
         const sessionToken = request.auth.credentials
         const service = request.payload.service || request.query.service
-        if (sessionToken.emailVerified && sessionToken.tokenVerified) {
-          return reply({})
-        }
-
-        // Choose which type of email and code to resend
-        let code, func, event
-        if (sessionToken.tokenVerificationId) {
-          code = sessionToken.tokenVerificationId
-        } else {
-          code = sessionToken.emailCode
-        }
+        var code, verifyFunction, event, emails = []
```

Comment:
why is there a `var` here? do we need a comment that specifies why we have to leave this as `var`?